applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -79,6 +79,19 @@ static void connected_cb(struct bt_conn *conn, uint8_t err)
 	uint8_t num_conn = 0;
 	struct bt_mgmt_msg msg;
 
+	if (err == BT_HCI_ERR_ADV_TIMEOUT && IS_ENABLED(CONFIG_BT_PERIPHERAL)) {
```

Comment:
So, timed out ADVs come back through the connected_cb?

Reply:
Correct